include/ListView/ListViewData.php

```diff
@@ -647,19 +647,19 @@ function getAdditionalDetails($fields, $adFunction, $editAccess)
 
 	        if($editAccess && !empty($results['editLink']))
 	        {
-	            $extra .=  "<a title=\'{$app_strings['LBL_EDIT_BUTTON']}\' href={$results['editLink']}><img style=\'margin-left: 2px;\' border=\'0\' src=\'".SugarThemeRegistry::current()->getImageURL('edit_inline.png')."\'></a>";
+	            $extra .=  "<a title=\'{$app_strings['LBL_EDIT_BUTTON']}\' href={$results['editLink']}><span class=\"suitepicon suitepicon-action-edit\"></span></a>";
```

Comment:
this `$extra` variable concatenated into an `<img>` tag `onclick` attribute which is a double quote marked (`"`) string in the generate html. You should escape the quotes properly if you don't want to close the onclick attribute, please double check the escaping is correct.

Reply:
I think you are referring to the href attribute? Yes? this is out of scope, I am only changing the images not how the code was implemented.

Reply:
No, I don't. I referring to the onclick attribute in the img tag as I wrote above. see https://github.com/salesagility/SuiteCRM/pull/5273/files/a96acab9993f625c531cc6b37bba312ecf392293#diff-1a31fc1ea6ff3d417c477289437da416L643